benchmark/result/metadata_class.py

```diff
@@ -0,0 +1,39 @@
+"""This module contains the metadata of a benchmark result."""
+
+from datetime import datetime
+
+import git
```

Comment:
It seems like this comes from a package called `GitPython`, do you need to add this to the installation?

Reply:
New dep group needs to be mentioned in relevant doc pages, i.e. Readme and there might also be a place in contributing or so...